applications/tests/test_applications_new_schema_queries.py

```diff
@@ -64,3 +65,118 @@ def test_berth_applications_no_customer_filter_false(berth_application, superuse
     executed = client.execute(query=query, graphql_url=GRAPHQL_URL, user=superuser)
 
     assert executed["data"] == {"berthApplications": {"edges": []}}
+
+
+def test_berth_applications_statuses_filter(berth_application, superuser):
+    berth_application.status = ApplicationStatus.HANDLED
+    berth_application.save()
+
+    client = GraphQLTestClient()
+    query = """
+        query APPLICATIONS {
+            berthApplications(statuses: [HANDLED]) {
+                edges {
+                    node {
+                        id
+                        status
+                    }
+                }
+            }
+        }
+    """
+    executed = client.execute(query=query, graphql_url=GRAPHQL_URL, user=superuser)
+
+    assert executed["data"] == {
+        "berthApplications": {
+            "edges": [
+                {
+                    "node": {
+                        "id": to_global_id(
+                            "BerthApplicationNode", berth_application.id
+                        ),
+                        "status": str(ApplicationStatus.HANDLED).upper(),
```

Comment:
could be `"status": ApplicationStatus.HANDLED.name,` to avoid string manipulations 😅